tests/parametric/test_otel_span_methods.py

```diff
@@ -483,9 +483,10 @@ def test_otel_span_reserved_attributes_overrides(self, test_agent, test_library)
         assert "span.type" not in span["meta"]
         assert "analytics.event" not in span["meta"]
 
-    @missing_feature(context.library < "java@1.25.0", reason="Implemented in 1.25.0")
+    @missing_feature(context.library == "java", reason="OTLP behavior not respected")
```

Comment:
This is not related to OTLP. We choose to not follow the golang boolean string parsing convention.

An honest reason would be "choose to not implement go parsing logic".

Reply:
Ok, will change this :+1:

Reply:
What about making a second test case with the extended parameters? So we can keep testing the expected behavior on the usual boolean (including using Java) and have a test covering the extended values.
As the tests is parametrize, it should be easy.

Reply:
Makes sense. I'd obviously prefer that we all have the same behavior, but yeah.

Reply:
In the end, it will still result of having a partial support feature dashboard for Java so it won't be unnoticed. 
Instead, you will just have a more granular evaluation.

Reply:
>We choose to not follow the golang boolean string parsing convention.

Same for Ruby, and I see that you have already updated the skip reason description @PROFeNoM so thank you!

I also don't foresee us making this test pass for the provided values in Ruby, given that:
* This specific boolean parsing is not consistent with other boolean environment variables at Datadog
* The full list of values that `strconv#ParseBool` considers true and false look pretty foreign to the Ruby environment.